world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
+
+#Create the skills database
+skill_manager = {}
+
+# hide skill
+skill_manager['hide'] = Ability()
+skill_manager['hide'].name = 'hide'
+skill_manager['hide'].type = 'skill'
+skill_manager['hide'].description = 'Allows a player to hide'
+skill_manager['hide'].level = 1
+skill_manager['hide'].hp_cost = 0
+skill_manager['hide'].mana_cost = 10
+skill_manager['hide'].move_cost = 0
+skill_manager['hide'].modifier = 0
+skill_manager['hide'].msg_to_caller = 'You slip into the shadows.'
+skill_manager['hide'].msg_to_room = '*caller* slips into the shadow.'
+skill_manager['hide'].msg_to_target = ''
+skill_manager['hide'].use_delay = 1
+skill_manager['hide'].affects = {}
+
+# kick skill
+skill_manager['kick'] = Ability()
+skill_manager['kick'].name = 'kick'
+skill_manager['kick'].type = 'skill'
+skill_manager['kick'].level = 1
+skill_manager['kick'].description = 'Allows a player to kick'
+skill_manager['kick'].hp_cost = 0
+skill_manager['kick'].mana_cost = 0
+skill_manager['kick'].move_cost = 10
+skill_manager['kick'].modifier = 0
+skill_manager['kick'].msg_to_caller = 'You kick *target*.'
+skill_manager['kick'].msg_to_room = '*caller* kicks *target*.'
+skill_manager['kick'].msg_to_target = '*caller* kicks you.'
+skill_manager['kick'].use_delay = 1
+skill_manager['kick'].affects = {}
+
+def show_skill_msg(mode=None,skill_name=None,caller=None,target=None):
+    """
```

Comment:
I tried this and it prevented me from starting up the server with the following error msg:

  File "D:\Github\ainneve\typeclasses\characters.py", line 16, in <module>
    from world.spells.spellmanager import spell_manager
  File "D:\Github\ainneve\world\spells\spellmanager.py", line 1, in <module>
    from world.abilities.ability import Ability
  File "D:\Github\ainneve\world\abilities\ability.py", line 1, in <module>
    from world.spells.spellmanager import spell_manager
ImportError: cannot import name spell_manager

It feels circular, as I have to import the skill and spell manager dictionary into the ability.py file.  And then this is reversed, as I have to also import Ability into skill and spellmanager.py files.

It won't let me start the server up.

Can you please help troubleshoot this and let me know what I'm doing wrong?

Also, does this mean we would need to execute this command using the following syntax:
     Ability().show_ability_msg('skills','kick','MSG_TO_CALLER',caller,None)
     Ability().show_ability_msg('spells','fireball','MSG_TO_TARGET',caller,target)